pkgs/cronet_http/tool/prepare_for_embedded.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
```

Comment:
I think that we need a big comment here to explain what this does. Like:

/// The `cronet_http` directory is used to produce two packages:
/// 1. `package:cronet_http`, which uses the Google Play Services version of Cronet
/// 2. `package:cronet_http_embedded`, which embeds Cronet
///
/// The default configuration of this code is to use the Google Play Services version of Cronet.
///
/// This script transforms the default configuration into one that embeds Cronet by:
/// 1. modifying the Gradle build file to reference the embedded version of Cronet
/// 2. modifying the `name` and `description` in `pubspec.yaml`
/// 3. replaying `README.md` with `README_EMBEDDED.md`
///
/// After running this script, `flutter pub publish` can be run to update `package:cronet_http_embedded`.
///
/// **NOTE:*** This script modifies the above files **in place**.